Misc/NEWS.d/next/Core and Builtins/2024-05-02-20-16-12.gh-issue-118524.B4rIYi.rst

```diff
@@ -0,0 +1 @@
+Since Python 3.10, it was possible to use unions as the second argument to ``isinstance``. Now, PEP604 style unions types, can also be used as match patterns.
```

Comment:
```suggestion
Since Python 3.10, it was possible to use unions as the second argument to :func:`isinstance`. Now, :pep:`604` style unions types, can also be used as match patterns.
```

Reply:
Drop the last, comma.